am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/DefaultRoleSetupRepository.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.amlib.repositories;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindBean;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import uk.gov.hmcts.reform.amlib.enums.AccessManagementType;
+import uk.gov.hmcts.reform.amlib.enums.RoleType;
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+import uk.gov.hmcts.reform.amlib.models.DefaultPermission;
+import uk.gov.hmcts.reform.amlib.models.ResourceAttribute;
+
+@SuppressWarnings({"LineLength", "PMD"})
+public interface DefaultRoleSetupRepository {
+    @SqlUpdate("insert into services (service_name, service_description) values (:serviceName, :serviceDescription)"
+        + " on conflict on constraint services_pkey do update set service_name = :serviceName, service_description = :serviceDescription")
+    void addService(@Bind("serviceName") String serviceName, @Bind("serviceDescription") String serviceDescription);
```

Comment:
Please remove bind, is not needed since we compile with parameter names. Binding should be derived from parameter names.

> Compiling with parameter names removes the need for the @Bind annotation. Jdbi will then bind each un-annotated parameter to the parameter’s name.

> `@SqlUpdate("insert into users (id, name) values (:id, :name)")
void insert(long id, String name);`

Reply:
I believe it is needed with the on conflict on constaint section